ckan/views/group.py

```diff
@@ -466,8 +466,10 @@ def read(group_type, is_organization, id=None, limit=20):
     # if the user specified a group id, redirect to the group name
     if data_dict['id'] == group_dict['id'] and \
             data_dict['id'] != group_dict['name']:
-        return h.redirect_to(u'{}.read'.format(group_type),
-                             id=group_dict['name'])
+        return h.redirect_to(
+            h.add_url_param(
+                controller=group_type, action=u'read',
```

Comment:
Could you change this to follow Flask practices for the url building as we are removing controllers and actions